src/components/preview_component/preview.sh

```diff
@@ -99,9 +99,13 @@ run_preview() {
     # if running end to end script with sbatch then use
     # sbatch to take advantage of multiple cores 
     if "$UseSlurm"; then
-        # set number of cores to run preview with
+        # set number of cores and memory to run preview with
         if "$isAWS"; then
-            sed -i -e "s:#SBATCH -c 8:#SBATCH -c ${cpu_count}:g" ${InversionPath}/src/inversion_scripts/imi_preview.py
+            sed -i -e "s:#SBATCH -c 8:#SBATCH -c ${cpu_count}:g" \
+                   -e "s:#SBATCH --mem:##SBATCH --mem:g" ${InversionPath}/src/inversion_scripts/imi_preview.py
+        else
+            sed -i -e "s:##SBATCH:#SBATCH:g" \
```

Comment:
Is this just in case the `imi_preview.py` script has `##` in the SBATCH header? Currently it just has single `#`.

Reply:
In the above case you can see that we comment out the memory if aws is true. I have the uncomment there in case a user accidentally had the wrong setting causing the batch command to be commented out.